lib/diyepw/setup.py

```diff
@@ -0,0 +1,12 @@
+import setuptools  # pragma: no cover
```

Comment:
This needs to call a `requirements.txt` file holding the package and version of utilized packages (see https://github.com/IMMM-SFA/im3py/blob/master/setup.py).  also needs to ensure that package data is installed as referenced in the MANIFEST.in file

Reply:
requirements.txt is added. MANIFEST.in seems like overkill to me since for our purposes here the same task can be accomplished with `package_data`

```
      package_data={ 'diyepw': ['data/*'] }
```

Note that I also moved the `files` directory to the package root and renamed it as requested elsewhere.